lib/join_cluster.js

```diff
@@ -346,19 +347,27 @@ JoinCluster.prototype.joinNode = function joinNode(node, callback) {
         incarnationNumber: this.ringpop.membership.localMember.incarnationNumber
     });
 
-    this.ringpop.channel.send(joinOpts, '/protocol/join', null, joinBody, function onSend(err, head, body) {
-        if (err) {
-            return callback(err, node);
-        }
+    console.log('how many calls ?!?!?!');
+    this.ringpop.channel
```

Comment:
interface changed to `.send().request()`

The callback changed to `(err, resp)` where `resp.ok` is true/false (like statusCode === 200).